src/plugins/rust/rust-parser-generator-trait.js

```diff
@@ -274,8 +274,13 @@ const RustParserGeneratorTrait = {
 
       lexRulesArray.push(`Tokenizer::_lex_rule${i}`);
 
-      // Example: r"\d+",
-      return `r"${flags}${rule.getRawMatcher()}"`;
+      let matcher = rule.getRawMatcher();
+      
+      // why there is f**king escape for / ?
```

Comment:
Let's use sensible comments 😄 Can you check please it works w/o escaping the `/`? Perhaps it was just copy-pasted from the original plugin for JS, and don't make sense for Rust.